components/commons/booking-form.tsx

```diff
@@ -115,7 +157,11 @@ const BookingForm: React.FC<BookingFormProps> = ({
           control={control}
           rules={{ required: true }}
           render={({ field }) => (
-            <TravelersInput onChange={field.onChange} />
+            <TravelersInput
+              onChange={field.onChange}
+              hasError={formState.errors.participants !== undefined}
```

Comment:
errors.participants is optional, no need to match if not undefined

`remove !== undefined`